### ISSUE TITLE
Revert #80; breaks arg splitting on OS X/Linux

### ISSUE THREAD

**Issue by sterpe** [mikeal/watch#85]
In 0.17 argument splitting is now broken on OSX/Linux when single quotes are used correctly.  

Basically: 

Given `package.json`:

```
"watch": "watch 'npm run css' ./src",
```

OSX/Linux
```
process.argv.slice(2) => ['npm run css', './src'] // OK
process.argv.slice(2).join(' ') => 'npm run css ./src' 
require('splitargs')(process.argv.slice(2).join(' ')) => [ 'npm', 'run', 'css', './src'] 

```
Please see discussion in #80.

**Comment by finnp** [comment]
Thank you for catching this! I just published 0.17.1

**Comment by sterpe** [comment]
No worries, love `watch`.  Keep up great work!
